docs/book/advanced-guide/gcp_entrypoint.md

```diff
@@ -0,0 +1,277 @@
+# Getting started with Google Cloud
+
+This guide will take you from the default local stack to an opinionated cloud 
+stack in no time. This should help you get started with running your machine 
+learning pipeline in production.
+
+
+{% hint style="info" %}
+This guide represents **one** of many ways to create a cloud stack on gcp. 
+Every component could be replaced by a different implementation.
+{% endhint %}
+
+## Prerequisites 
+
+For this to work you need to have zenml installed locally with all gcp 
+requirements.
+
+```bash
+pip install zenml
+zenml integration install gcp
+```
+
+## The cloud stack
+
+A full cloud stack will necessarily contain these five stack components:
+
+* An **artifact store** to save all step output artifacts, in this guide we will
+use a gcp bucket for this purpose
+* A **metadata store** that keeps track of the relationships between artifacts, 
+runs and parameters. In our case we will opt for a MySQL database on gcp
+Cloud SQL.
+* The **orchestrator** to run the pipelines. Here we will opt for a Vertex AI
+pipelines orchestrator. This is a serverless GCP specific offering with minimal
+hassle.
+* A **container registry** for pushing and pulling the pipeline container.
+* Finally, the **secret manager** to store passwords and ssl certificates.
+
+## Step 1/10 Set up a gcp project (Optional)
+
+As a first step it might make sense to 
+[create](https://console.cloud.google.com/projectcreate) 
+a separate gcp project for your zenml resources. However, this step is 
+completely optional, and you can also move forward within an existing project. 
+If some resources already exist, feel free to skip the creation of the 
+resources and simply note down the relevant information. 
+
+For simplicity, just
+open up a terminal on the side and export relevant values as we go along. You
+will use these when we set up the ZenML stack.
+ZenML will use your project ID at a later stage to connect to some 
+resources, so let's export it. You'll most probably find it right 
+[here](https://console.cloud.google.com/welcome)
+
+```bash
+export PROJECT_ID=... 
```

Comment:
Would be nice to give an example of this as people often confuse the ID with the name of the project

Reply:
I feel like it's best if we are consistent with these, I would suggest the following:
If people should replace something with their own values, we use the `<placeholder>` syntax that we decided on.
If we have an example value, put it in a comment:
```bash
export PROJECT_ID=<PROJECT_ID>  # for example ...
```

Reply:
Another thing regarding this guide: All these `exports` won't work on Windows I think. I personally don't mind it, just something to be aware of when we write these.

Reply:
oh just saw this comment - replaced all of them with concrete examples, but I am concerend that these values will not be replaced. Any way we coul use placeholders but have examples of what it could look like as well?

Reply:
Have the example value as comments potentially, or maybe in the text above?

Reply:
@schustmi should I offer a windows tab that shows the same thing in windows-lingo:

```
$Env:DB_PWD = 'example_pwd'
```

Reply:
IMO no, far too much work. But maybe @htahir1 disagrees, I would write all our guides unix only